lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -154,6 +154,12 @@ def publishing_api_has_item(item)
         stub_request(:get, url).to_return(status: 200, body: item.to_json, headers: {})
       end
 
+      def publishing_api_has_links(links)
+        links = links.with_indifferent_access
+        url = PUBLISHING_API_V2_ENDPOINT + "/links/" + links[:content_id]
```

Comment:
I discussed this with @danielroseman and he said to follow the approach I've taken. For reference, the `links` payload looks like this:

```json
{
  "content_id": "b1dc075f-d946-4bcb-a5eb-941f8c8188cf",
  "links": {
    "organisations": [
      "d39237a5-678b-4bb5-a372-eb2cb036933d"
    ]
  },
  "version": 193
}
```